cypress-tests/cypress/support/pageBuilder/pbDeleteAllCategories.ts

```diff
@@ -0,0 +1,70 @@
+import { gqlClient } from "../utils"; // Replace with your actual gqlClient library
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            pbDeleteAllCategories(): Promise<void | any[]>;
```

Comment:
```suggestion
            pbDeleteAllCategories(): Promise<void[]>;
```

`Promise<void>` is not the correct so I removed it. Let's try avoiding `any`.